
---
std_msgs/Bool finished
CollisionInformation ci